app/web/features/communities/CommunityPage/CommunityPage.tsx

```diff
@@ -101,7 +101,7 @@ export default function CommunityPage({
             ) : tab === "discussions" ? (
               <DiscussionsListPage community={community} />
             ) : tab === "events" ? (
-              <EventsList community={community} />
+              <CommunityEventsList community={community} />
```

Comment:
We use the old style events list on the community page. Maybe we eventually redesign that to be new, but for now I just rename the old one to keep it looking the same (this feature is already big enough).